core/controllers/classifier.py

```diff
@@ -107,7 +107,14 @@ def post(self):
             raise self.UnauthorizedUserException
 
         job_id = message['job_id']
-        classifier_data = message['classifier_data']
+        # The classifier data received in the payload has all floating point
```

Comment:
Link to https://stackoverflow.com/q/40173295 here

Reply:
Done.